shub/config.py

```diff
@@ -350,12 +350,18 @@ def get_image(self, target):
                 "Could not find image for project '{}'. Please define it "
                 "in your scrapinghub.yml.".format(target))
         elif image is False:
```

Comment:
```
projects:
  default: 123
  stackbased:
    id: 456
    image: false
    stack: scrapy:1.3

image: true
```

is probably a more real world example of why the separation is necessary ;) `shub image upload stackbased` will tell the user "using custom image is disabled for target 'stackbased'" instead of "could not find image"